CHANGELOG.md

```diff
@@ -1,33 +1,68 @@
 # Change Log
 
-## [1.8.0](https://github.com/test-kitchen/test-kitchen/tree/1.8.0) (2016-05-05)
-[Full Changelog](https://github.com/test-kitchen/test-kitchen/compare/v1.7.3...1.8.0)
+## [1.9.0](https://github.com/test-kitchen/test-kitchen/tree/1.9.0) (2016-05-26)
+[Full Changelog](https://github.com/test-kitchen/test-kitchen/compare/v1.8.0...1.9.0)
+
+**Implemented enhancements:**
+
+- Cannot set log level to "info" [\#529](https://github.com/test-kitchen/test-kitchen/issues/529)
+- Buffer errors until the end of an action [\#1034](https://github.com/test-kitchen/test-kitchen/pull/1034) ([smurawski](https://github.com/smurawski))
+
+**Fixed bugs:**
+
+- Use command\_prefix provided by Kitchen::Provisioner::Base in shell provisioner [\#1033](https://github.com/test-kitchen/test-kitchen/pull/1033) ([pstengel](https://github.com/pstengel))
```

Comment:
I'd say it was a bug - command prefix should have been supported there when it was implemented, but I'm not super invested in that opinion.